lib/Server.js

```diff
@@ -31,7 +33,6 @@ const getCompilerConfigArray = require('./utils/getCompilerConfigArray');
 const getStatsOption = require('./utils/getStatsOption');
 const getColorsOption = require('./utils/getColorsOption');
 const setupExitSignals = require('./utils/setupExitSignals');
-const findPort = require('./utils/findPort');
```

Comment:
Forgot one note - we should keep this file due https://github.com/webpack/webpack-cli/blob/webpack-cli%404.6.0/packages/serve/src/startDevServer.ts and old other verions with todo to remove it in the next major release and reexport static method

Reply:
oh yep lets keep it